docs/Reference/ViewManipulation.md

```diff
@@ -99,7 +99,7 @@ Creates a new view with a given name and properties if it does not already exist
 
   The name of the view.
 
-- **properties**: `object` (optional)
+- **links**: `object` (optional)
 
     - **links**:
```

Comment:
you are already taking the link object so no need to take an object with links object in it. just use direct object like primary sort.